console/src/main/java/org/georchestra/console/ws/editorgdetails/EditOrgDetailsFormController.java

```diff
@@ -108,6 +110,7 @@ public String edit(Model model, @ModelAttribute EditOrgDetailsFormBean formBean,
         validation.validateOrgField("url", formBean.getUrl(), resultErrors);
         validation.validateOrgField("address", formBean.getAddress(), resultErrors);
         validation.validateOrgField("description", formBean.getDescription(), resultErrors);
+        // TODO validate mail address for the organization ?
```

Comment:
which kind of validations are performed on the other fields here ? Would it be relevant for the email too ?